deploy/requirements.yml

```diff
@@ -17,28 +17,28 @@
 # Can be cleared when this is closed: https://github.com/pmbauer/ansible-tarsnap/pull/3
 - name: pmbauer.tarsnap
   src: https://github.com/open-craft/ansible-tarsnap
-  version: d96b1e69ae6b930e0841d7b025a3070c408a23f4
+  version: origin/master
```

Comment:
We can't use master for this one.  It's a fork of a third-party role, and the master branch is the upstream master at the time of forking, and not the branch containing our modification.

Sorry, I didn't think of this one when I said "all roles owned by the open-craft org".